libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/LanguageGenerator.cs

```diff
@@ -17,7 +17,7 @@ public abstract class LanguageGenerator
         /// <param name="dialogContext">dialogContext.</param>
         /// <param name="template">template or [templateId].</param>
         /// <param name="data">data to bind to.</param>
-        /// <param name="cancellationToken">the <see cref="CancellationToken"/> for the task.</param>
+        /// <param name="cancellationToken">the <see cref="CancellationToken"/> that can be used by other objects or threads to receive notice of cancellation.</param>
```

Comment:
Is this a change from previous comments? we should only add missing documentations